dashboard/app/models/ability.rb

```diff
@@ -132,6 +133,7 @@ def initialize(user)
         can :manage, :pd_workshop_summary_report
         can :manage, :pd_teacher_attendance_report
         can :manage, Pd::TeacherApplication
+        can :manage, :pd_workshop_user_admin
```

Comment:
minor: this name `user_admin` implies to me that we're editing admins. Maybe it (and the controller) would be better named something like `pd_workshop_user_permissions`?